### ISSUE TITLE
Nexus Question

### ISSUE THREAD

**Issue by berkhornet** [kodi-pvr/pvr.iptvsimple#656]
Is there a way for a skin to display the channel provider name via an Infolabel? Many thanks

**Comment by phunkyfish** [comment]
Yes, it’s should be under ProviderName for channel, recording and probably timer if available.

**Comment by berkhornet** [comment]
Thanks, I will experiment. Haven't found the correct Infolabel yet in the v20 documentation. I tried $INFO[ListItem.ProviderName] but that didn't return anthing.

**Comment by berkhornet** [comment]
I haven’t found an answer - perhaps the provider information is not available. When playing an IPTV channel, and pressing “O” shows provider as “Unknown”
